application-tour-ui/src/main/resources/TourCode/TourJS.xml

```diff
@@ -276,8 +274,6 @@ require(['jquery', 'xwiki-meta'], function ($, xm) {
       tour.init();
       if (forceStart) {
         tour.restart();
-      } else {
-        tour.start();
```

Comment:
This is breaking the start of tours, and this is one reason the test is failing at http://ci.xwiki.org/job/XWiki%20Contrib/job/application-tour/job/master/2/testReport/org.xwiki.contrib.tour.test.ui/TourApplicationTest/testTour/

I've reopened the jira issue at http://jira.xwiki.org/browse/TOUR-52